torch/_inductor/kernel/mm.py

```diff
@@ -570,6 +570,13 @@ def lazy_register_extern_choice(fn):
 
 aten_mm = ExternKernelChoice(torch.mm, "at::mm_out")
 
+aten_mm_dtype = ExternKernelChoice(
+    torch.mm,
+    "at::_mm_dtype_out_cuda",
```

Comment:
```suggestion
    "at::mm_dtype_out",
```
This will make cpp_wrapper/AOTI happy.

Reply:
And you need to bring back your change to `fallback_ops.py`.